src/cmds/categories/create.ts

```diff
@@ -13,60 +12,34 @@ interface Category {
   type: string;
 }
 
-interface Options {
-  categoryType?: 'guide' | 'reference';
-  preventDuplicates?: boolean;
-  title?: string;
-}
-
-export default class CategoriesCreateCommand extends Command {
-  constructor() {
-    super();
-
-    this.command = 'categories:create';
-    this.usage = 'categories:create <title> [options]';
-    this.description = 'Create a category with the specified title and guide in your ReadMe project.';
-    this.cmdCategory = CommandCategories.CATEGORIES;
-
-    this.hiddenArgs = ['title'];
-    this.args = [
-      this.getKeyArg(),
-      this.getVersionArg(),
-      {
-        name: 'title',
-        type: String,
-        defaultOption: true,
-      },
-      {
-        name: 'categoryType',
-        type: String,
-        description: 'Category type, must be `guide` or `reference`',
-      },
-      {
-        name: 'preventDuplicates',
-        type: Boolean,
-        description:
-          'Prevents the creation of a new category if there is an existing category with a matching `categoryType` and `title`',
-      },
-    ];
-  }
-
-  async run(opts: AuthenticatedCommandOptions<Options>) {
-    await super.run(opts);
-
-    const { categoryType, title, key, version, preventDuplicates } = opts;
-
-    if (!title) {
-      return Promise.reject(new Error(`No title provided. Usage \`${config.cli} ${this.usage}\`.`));
-    }
-
-    if (categoryType !== 'guide' && categoryType !== 'reference') {
-      return Promise.reject(new Error('`categoryType` must be `guide` or `reference`.'));
-    }
+export default class CategoriesCreateCommand extends BaseCommand<typeof CategoriesCreateCommand> {
+  static description = 'Create a category with the specified title and guide in your ReadMe project.';
+
+  static args = {
+    title: Args.string({ description: 'Title of the category', required: true }),
+  };
+
+  static flags = {
+    categoryType: Flags.option({
+      description: 'Category type',
+      options: ['guide', 'reference'] as const,
+      required: true,
+    })(),
+    key: keyFlag,
+    preventDuplicates: Flags.boolean({
+      description:
+        'Prevents the creation of a new category if there is an existing category with a matching `categoryType` and `title`',
```

Comment:
Definitely doesn't need to happen here but before we publish a v9 with APIv2 support we should do a full pass on all flag and argument descriptions, the punctuation and terminology in some of them could use some consistency passes.

Reply:
totally agreed! i'm planning on doing a deep revamp of our docs (including a migration guide) prior to release